Validation/RecoTrack/scripts/harvestTrackValidationPlots.py

```diff
@@ -0,0 +1,52 @@
+import sys,os,tempfile,shutil,subprocess,glob
+import argparse
+
+if __name__ == "__main__":
+
+    # define options
+    parser = argparse.ArgumentParser(description="Harvest track validation plots")
+    parser.add_argument("files", metavar="file", type=str, nargs="+",
+                        help="files to be harvested (convert edm DQM format to plain ROOT format")
+    parser.add_argument("-o", "--outputFile", type=str, default="harvest.root",
+                        help="output file (default: 'harvest.root')")
+
+    opts = parser.parse_args()
+
+    # absolute path outputFile
+    outputFile = os.path.abspath(opts.outputFile)
+
+    # check the input files
+    for f in opts.files:
+        if not os.path.exists(f):
+            parser.error("DQM file %s does not exist" % f)
+
+    # compile a file list for cmsDriver
+    filelist = ",".join(["file:{0}".format(os.path.abspath(_file)) for _file in opts.files])
+
+    # go to a temporary directory
+    _cwd = os.getcwd()
+    _tempdir = tempfile.mkdtemp()
+    os.chdir(_tempdir)
+
+    # compile cmsDriver command
+    cmsDriverCommand = "cmsDriver.py harvest --scenario pp --filetype DQM --conditions auto:run2_mc --mc -s HARVESTING:postProcessorTrackSequence -n -1 --filein {0}".format(filelist)
+    print "# running cmsDriver" + "\n" + cmsDriverCommand
+    
+    # run it
+    subprocess.call(cmsDriverCommand,shell=True)
```

Comment:
Running
```
$ python
>>> cmsDriverCommand = "cmsDriver.py harvest --scenario pp --filetype DQM --conditions auto:run2_mc --mc -s HARVESTING:postProcessorTrackSequence -n -1 --filein foo.root --no_exec"
>>> subprocess.call(cmsDriverCommand.split(" "))
HARVESTING:postProcessorTrackSequence
entry foo.root
...
Config file harvest_HARVESTING.py created
0
>>>
```
works for me in 750pre5 (and I remembered wrongly, by default subprocess.call does redirect the output of the command to stdout instead of hiding it).